### ISSUE TITLE
Display problem with long lines on Math Notepad

### ISSUE THREAD

**Issue by ldecarufel** [josdejong/mathjs#3326]
Hello! I hope this is the right plate to post these kind of issues.

I've been a **SpeQ** user for several years, and I'm now trying to use it's successor **Math Notepad**.

However, I have some display issues with long lines.
Basically, it would be nice if they were displayed on multiple lines.

Currently, we only see either the end of the code line, or the start.  This makes it really difficult to read the sheet as a whole.

Seeing start of line:
![image](https://github.com/user-attachments/assets/b48cdd5d-7708-49cb-b697-1d7c536a398b)

Or seeing the end of the line:
![image](https://github.com/user-attachments/assets/e533fe9a-454e-4af2-b083-27c524087067)

It would be nice if the page width could be enlarged, or if long formulas could be displayed on multiple lines.
Also, it would be nice if "..." would be displayed to show that the line is not complete and some part is hidden.

**Comment by josdejong** [comment]
Thanks for your suggestion Louis. The editor indeed has an issue with showing long lines that are wrapped.

What you can do though is spread your long formulas over multiple lines by pressing <kbd>Shift+Enter</kbd> to create a new line in the current expression

**Comment by josdejong** [comment]
Yes, one idea I have is that you can import other files, and I would like to set up a "marked place" where you can share usable collections with functions.
